### ISSUE TITLE
Can not install on laravel 5.8 with jenssegers/mongodb 3.5

### ISSUE THREAD

**Issue by tuyenlaptrinh** [designmynight/laravel-mongodb-passport#21]
This is error when I was trying install on Laravel 5.8.11 with jenssegers/mongodb 3.5
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - designmynight/laravel-mongodb-passport v1.1.5 requires jenssegers/mongodb 3.3.* || 3.4.* -> satisfiable by jenssegers/mongodb[v3.3.0, v3.3.0-alpha, v3.3.1, v3.4.0, v3.4.1, v3.4.2, v3.4.3, v3.4.4, v3.4.5, v3.4.6] but these conflict with your requirements or minimum-stability.
    - designmynight/laravel-mongodb-passport v1.1.4 requires jenssegers/mongodb 3.3.* || 3.4.* -> satisfiable by jenssegers/mongodb[v3.3.0, v3.3.0-alpha, v3.3.1, v3.4.0, v3.4.1, v3.4.2, v3.4.3, v3.4.4, v3.4.5, v3.4.6] but these conflict with your requirements or minimum-stability.
    - designmynight/laravel-mongodb-passport v1.1.3 requires jenssegers/mongodb 3.3.* || 3.4.* -> satisfiable by jenssegers/mongodb[v3.3.0, v3.3.0-alpha, v3.3.1, v3.4.0, v3.4.1, v3.4.2, v3.4.3, v3.4.4, v3.4.5, v3.4.6] but these conflict with your requirements or minimum-stability.
    - designmynight/laravel-mongodb-passport v1.1.2 requires jenssegers/mongodb 3.3.* || 3.4.* -> satisfiable by jenssegers/mongodb[v3.3.0, v3.3.0-alpha, v3.3.1, v3.4.0, v3.4.1, v3.4.2, v3.4.3, v3.4.4, v3.4.5, v3.4.6] but these conflict with your requirements or minimum-stability.
    - designmynight/laravel-mongodb-passport v1.1.1 requires jenssegers/mongodb 3.3.* -> satisfiable by jenssegers/mongodb[v3.3.0, v3.3.0-alpha, v3.3.1] but these conflict with your requirements or minimum-stability.
    - designmynight/laravel-mongodb-passport v1.1.0 requires jenssegers/mongodb 3.3.* -> satisfiable by jenssegers/mongodb[v3.3.0, v3.3.0-alpha, v3.3.1] but these conflict with your requirements or minimum-stability.
    - Installation request for designmynight/laravel-mongodb-passport ^1.1 -> satisfiable by designmynight/laravel-mongodb-passport[v1.1.0, v1.1.1, v1.1.2, v1.1.3, v1.1.4, v1.1.5].


Installation failed, reverting ./composer.json to its original content.
```
I hope owner can fix it soon.
Thank you

**Comment by Siebov** [comment]
Any updates?

**Comment by andrycs** [comment]
https://github.com/designmynight/laravel-mongodb-passport/pull/22
this problem is caused by  Invalid version string "dev*" in laravel passport dependencies on composer.json 
please merge this PR or use a forked one

thankss!

also please submit the latest version on the packagist, thanks!

**Comment by matiaspub** [comment]
that's can help while we waiting
https://www.sitepoint.com/quick-tip-composer-github-develop-packages-interactively/

**Comment by matiaspub** [comment]
Duplicate of #19

**Comment by dungfv** [comment]
This package seems to be no longer supported

**Comment by matiaspub** [comment]
@hoatructhanhlam https://github.com/votong/laravel-mongodb-passport seems work correctly

**Comment by jmosul** [comment]
> #22
> this problem is caused by Invalid version string "dev*" in laravel passport dependencies on composer.json
> please merge this PR or use a forked one
> 
> thankss!
> 
> also please submit the latest version on the packagist, thanks!

We've removed the *dev version. Sorry for the delay, but now released in version 1.1.7
